docker/Dockerfile-dhvirtualenv

```diff
@@ -14,32 +14,29 @@ ARG distro=""
 
 ###
 ### Stage 0: build a dh-virtualenv
+### Based on https://github.com/spotify/dh-virtualenv/blob/master/Dockerfile
 ###
+
 FROM ${distro} as builder
 
-RUN apt-get update -qq -o Acquire::Languages=none
-RUN env DEBIAN_FRONTEND=noninteractive apt-get install \
-        -yqq --no-install-recommends \
-        build-essential \
-        ca-certificates \
-        devscripts \
-        equivs \
-        wget
+RUN apt-get update -qq -o Acquire::Languages=none \
+    && env DEBIAN_FRONTEND=noninteractive apt-get install \
+        -yqq --no-install-recommends -o Dpkg::Options::=--force-unsafe-io \
```

Comment:
I'm giving `--force-unsafe-io` some side-eye. The warnings on `dpkg` sound a bit dire: are you sure it is safe?

Reply:
I see that dh-virtualenv's Dockerfile does it, and that we already do it below, so :man_shrugging: I guess - though I'd like to know if it is written down anywhere that this is an ok thing to do when building images.

Reply:
It's totally fine. All it does is disable fsync calls, which can be a problem if a real system with a persistent disk crashes midway through installation, but in a Docker container we either succeed and snapshot filesystem in its entirety a new layer, or we fail and everything goes away.

Reply:
fair enough!